src/mlpack/tests/linear_svm_test.cpp

```diff
@@ -543,8 +548,11 @@ BOOST_AUTO_TEST_CASE(LinearSVMLBFGSTwoClasses)
       labels(i) = 1;
     }
 
+    CallbackTestFunction cb;
+    ens::L_BFGS opt;
     // Create a linear svm object using L-BFGS optimizer.
-    LinearSVM<arma::mat> lsvm(data, labels, numClasses, lambda);
+    LinearSVM<arma::mat> lsvm(data, labels, numClasses, lambda,
```

Comment:
I think it's fine to test the model without any callbacks, as it should work as well and should also be tested, so do you mind to revert the changes here?